packages/gatsby-cli/README.md

```diff
@@ -37,6 +37,8 @@ development server.
 | `-o`, `--open`  | Open the site in your (default) browser for you |
 | `-S`, `--https` | Use HTTPS                                       |
 
+With `-o` flag enabled, for MacOS Chrome user, we will try our best to reuse the same tab. For any other scenarios, it will fallback to open a new tab instead.
```

Comment:
I would just remove this, perhaps. It's almost not worth documenting the "edge" case, and the dev will just get that experience when using?

Reply:
@DSchau Make sense to me, I guess most people won't even read this file.